cfme/tests/webui/test_advanced_search.py

```diff
@@ -198,8 +201,14 @@ def test_filter_crud(param, appliance):
     filter_value_updated = fauxfactory.gen_string('alphanumeric', 10)
     view = _navigation(param, appliance)
     # create
-    view.search.save_filter(
-        "fill_field({}, =, {})".format(param.filter, filter_value), filter_name)
+    if ':' not in param.filter:  # to test "Count of" field, values don't contain ':'
+        filter_value = fauxfactory.gen_numeric_string(3)
+        filter_value_updated = fauxfactory.gen_numeric_string(3)
```

Comment:
question:
is there any character limit? it's the same as `L201`

Reply:
Not sure I understand the question correctly... It's not the same because of `gen_numeric_string`. When testing "Count of" an integer should be passed.

Reply:
aaaaahh you want numeric :+1: